Week-3/a2/index.js

```diff
@@ -0,0 +1,43 @@
+/*
+This is a common object for any software. It installs and confirms installion.
+*/
+var software = {
```

Comment:
@mr-rego prefer using let, const and not var. It looks good. Nice work. But i could see unnecessary files available from previous weeks. Can you merge those start working ?